src/lib/init.ts

```diff
@@ -5,12 +5,14 @@ import { filesystemStore, sessionStore } from '../stores'
 import { getBackupStatus, type BackupStatus } from '$lib/auth/backup'
 import { USERNAME_STORAGE_KEY } from '$lib/auth/account'
 
+export const NAMESPACE = { creator: 'Fission', name: 'WAT' }
```

Comment:
It might be worth moving this to `src/lib/app-info.ts` to make it easier for folks to change in their apps. Also worth mentioning it in the README in the "Rename your application" section.

Reply:
good call!

Reply:
done!